packages/ui/components/ui/tx/view/swap/index.tsx

```diff
@@ -53,7 +40,9 @@ export const SwapViewComponent = ({ value }: { value: SwapView }) => {
               )}
 
               <ActionDetails.Row label='Fee'>
-                <ValueViewComponent view={claimFeeValueView} />
+                <div className='font-mono'>
+                  {joinLoHiAmount(getAmount(claimFee)).toString()} upenumbra
+                </div>
```

Comment:
How it looks now
<img width="825" alt="image" src="https://github.com/penumbra-zone/web/assets/25391690/645bbdd3-2339-4541-81fc-9c7a0551d70e">